operator/cmd/mesh/install.go

```diff
@@ -175,6 +185,18 @@ func Install(rootArgs *RootArgs, iArgs *InstallArgs, logOpts *log.Options, stdOu
 	// "no running Istio pods in istio-system" for the first time
 	_ = detectIstioVersionDiff(p, tag, ns, kubeClient, setFlags)
 
+	if iArgs.ShowDiff {
+		diff, err := compareIOPWithInstalledIOP(kubeClient, iop)
+		if err != nil && !errors.IsNotFound(err) {
+			l.LogAndErrorf("Error when comparing installed IOP with the new one: %v", err)
+		} else if diff != "" {
+			l.LogAndPrint("The following differences were found when comparing the installed IOP with the new one:")
+			l.LogAndPrint(diff)
+		} else if diff == "" && !errors.IsNotFound(err) {
```

Comment:
Not sure, do we ever get here? Isn't the first if on line 190 catching the same error?

Reply:
line 190 will not catch IsNotFound error